dags/multipod/maxtext_end_to_end.py

```diff
@@ -38,7 +38,7 @@
   test_models = {
       "llama2": ["test_llama2", "llama_finetuning_test"],
       "mistral": ["test_mistral"],
-      "gamma": ["test_gamma"],
+      "gamma": ["test_gemma"],
```

Comment:
Should the key of this dict also be gemma?

Reply:
Fixed